demos/espressif/esp32_devkitc_esp_wrover_kit/make/partitions_example.csv

```diff
@@ -3,6 +3,6 @@
 nvs,      data, nvs,     0x10000,  0x6000
 otadata,  data, ota,     0x16000,  0x2000
 phy_init, data, phy,     0x18000,  0x1000
-ota_0,    0,    ota_0,   0x20000,  1M
-ota_1,    0,    ota_1,   ,         1M
-storage,  data, spiffs,  ,         0x10000
+ota_0,    0,    ota_0,   0x20000,  1500K
```

Comment:
Not new to this commit, but why are we leaving a 0x7000 gap between 0x19000 and 0x20000?  Is this on purpose?  I suppose it is not doing any harm, but if someone wanted to add another section, they might not notice that there was empty space here

Reply:
tbh I don't know. Need to be checked

Reply:
what about the other 0x6000?

Reply:
If user enables hardware security features (like secure boot, flash encryption) then bootloader can have larger footprint, so sufficient headroom (0x6000) is kept reserved such that application offset need not change going ahead.